### DISCUSSION TITLE
How could I find the current epoch number in the code?

### DISCUSSION THREAD

**Discussion by EdyQiu** [open-mmlab/mmclassification#1494] [General] [answered]
Hi, I want to add an auxiliary loss in the head module, and it is designed to start working when the epoch number > 5.
Therefore I want to ask how could I know the current epoch number, is there any function for that?
Thanks for help!

**Comment by Ezra-Yu** [comment, marked as answer]
Based on your needs, it is highly recommended that you use [SwitchRecipeHook](https://github.com/open-mmlab/mmpretrain/blob/e80418a424aaefb81c95df458216bb3e9af246c4/mmpretrain/engine/hooks/switch_recipe_hook.py#L14).

```
loss_after_30e = [dict(type='CE'), dict(type='Mine')]
loss_after_60e = [dict(type='CE')]
custom_hooks = [
    dict(
        type='SwitchRecipeHook',
        schedule=[
            dict(action_epoch=30, loss=loss_after_30e),
            dict(action_epoch=60, loss=loss_after_60e),
        ]
    )
]
```
